src/Microsoft.DotNet.Build.Tasks/PackageFiles/FrameworkTargeting.targets

```diff
@@ -105,6 +105,10 @@
   <Target Name="ConvertCommonMetadataToAdditionalProperties" BeforeTargets="AssignProjectConfiguration">
     <!-- list each append as a seperate item to force re-evaluation of AdditionalProperties metadata -->
     <ItemGroup>
+      <ProjectReference>
```

Comment:
This is required to make VS builds succeed. Without this System.Runtime.Tests fails with E:\corefx\Tools\Packaging.targets(589,5): error : Packages that are constrained by runtime should not have runtime dependencies.  They will be ignored by nuget